### PR TITLE
update clickhouse-jdbc version 0.5.0

### PR DESCRIPTION
**Pull request by mzitnik**

### CODE REVIEW

File: docs/en/integrations/language-clients/java/index.md [hunk 1]
```diff
@@ -25,7 +25,7 @@ Provides the most flexible and performant way to integrate your app with ClickHo
 
 | Client version | ClickHouse  |
 |----------------|-------------|
-| 0.4.6          | 20.7+       |
+| 0.5.0          | 20.7+       |
```
Comment (mshustov) [inline docs/en/integrations/language-clients/java/index.md:28]: Did we test with such an old version of ClickHouse? v23.3 is the last supported version of ClickHouse https://github.com/ClickHouse/ClickHouse/blob/master/SECURITY.md

**Comment by robot-clickhouse** [comment]
*This is an automated comment for commit ca261270bd93de28b2dbe524a8ec58b6d80f9b02 with description of existing statuses. It's updated for the latest CI running*

[✅ Click here](https://s3.amazonaws.com/clickhouse-test-reports/1580/ca261270bd93de28b2dbe524a8ec58b6d80f9b02/ci_running.html) to open a full report in a separate page

<details><summary>Successful checks</summary>
<table>
<thead><tr><th>Check name</th><th>Description</th><th>Status</th></tr></thead>
<tbody>
<tr><td>Docs Check</td><td>Builds and tests the documentation</td><td><a href="https://s3.amazonaws.com/clickhouse-test-reports/1580/ca261270bd93de28b2dbe524a8ec58b6d80f9b02/docs_check.html">✅ success</a></td></tr>
<tr><td>Vercel</td><td>There's no description for the check yet, please add it to tests/ci/ci_config.py:CHECK_DESCRIPTIONS</td><td><a href="https://vercel.com/clickhouse/clickhouse-docs/HJ1zULYA9bmxdtKnozzSjyRrCBL3">✅ success</a></td></tr>
<tbody>
</table>
</details>

File: docs/en/integrations/language-clients/java/index.md (hunk 1 above)
Comment (mzitnik) [reply docs/en/integrations/language-clients/java/index.md:28]: No missed it. I will change it

Comment (mzitnik) [reply docs/en/integrations/language-clients/java/index.md:28]: I will update with 22.8 since we run tests with that version

**Pull request merged**